languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -8479,6 +8479,42 @@ USA
         </rulegroup>
 
 
+       <rulegroup id='DAR_EXEMPLO_EXEMPLIFICAR' name="Simplificar: 'Dar exemplo' → exemplificar" tone_tags="academic" is_goal_specific="true" default="temp_off">
+
+            <!-- #1 - Without d[ao]s? -->
+            <rule>
+                <pattern>
+                    <marker>
+                        <token inflected='yes'>dar</token>
+                        <token regexp='yes'>um|uns|algum|alguns</token>
+                        <token regexp='yes'>exemplos?</token>
+                    </marker>
+                    <token postag='_PUNCT|SPS00|CC|CS' postag_regexp='yes'/>
+                </pattern>
+                <message>Num contexto formal/académico, é preferível escrever &quot;exemplificar&quot;.</message>
```

Comment:
We need messages to be dialect-neutral. Can we remove `académico` from here, as it doesn't match the pt-BR spelling?